server/protocols/jmap-rfc-8621/doc/specs/spec/mail/message.mdown

```diff
@@ -639,15 +662,27 @@ and response:
 
 ## Email/changes
 
+<aside class="warning">
+  Not implemented yet
+</aside>
+
 This is a standard "/changes" method as described in [@!RFC8620], Section 5.2. If generating intermediate states for a large set of changes, it is recommended that newer changes be returned first, as these are generally of more interest to users.
 
 ## Email/query
 
+<aside class="warning">
+    Partially implemented
+</aside>
+
 This is a standard "/query" method as described in [@!RFC8620], Section 5.5, but with the following additional request arguments:
 
 - **collapseThreads**: `Boolean` (default: false)
   If `true`, Emails in the same Thread as a previous Email in the list (given the filter and sort order) will be removed from the list. This means only one Email at most will be included in the list for any given Thread.
 
+<aside class="warning">
+  Each message belonging to a single thread according to our current implementation, collapseThread is a noop operation.
+</aside>
+
```

Comment:
we can add that the calcultateTotal is not supported and so nor are the negative position